logstash-output-amazon_es.gemspec

```diff
@@ -4,7 +4,7 @@
 
 Gem::Specification.new do |s|
   s.name            = 'logstash-output-amazon_es'
-  s.version         = '7.1.0'
+  s.version         = '7.1.1'
```

Comment:
As this change is making a fairly significant update to the AWS SDK, I think this should be a minor version bump (as opposed to a patch).

Can you change this to `7.2.0`?

Reply:
done